tests/ui/const-generics/generic_const_exprs/unify-op-with-fn-call.rs

```diff
@@ -1,3 +1,5 @@
+// known-bug: #110395
```

Comment:
sorry for the necro:

This is a bad annotation. This test no longer tests what it was intended to. Also, #110395 isn't a bug...